packages/openapi/test/decorators.test.ts

```diff
@@ -46,6 +47,74 @@ describe("openapi: decorators", () => {
   });
 
   describe("@extension", () => {
+    it.each([
+      ["minProperties", 1],
+      ["maxProperties", 1],
+      ["uniqueItems", true],
+      ["multipleOf", 1],
+    ])("apply extension on model prop with %s", async (key, value) => {
+      const { prop } = await runner.compile(`      
+          model Foo {
+            @extension("${key}", ${value})
+            @test
+            prop: string[]
+          }
+        `);
+
+      deepStrictEqual(Object.fromEntries(getSchemaExtensions(runner.program, prop)), {
+        [key]: value,
+      });
+      deepStrictEqual(Object.fromEntries(getExtensions(runner.program, prop)), {});
+    });
+
+    it.each(["minProperties", "maxProperties", "uniqueItems", "multipleOf"])(
```

Comment:
Looks like they are focusing on testing array as target. please note that they applies to different types, please add test for different types. FYI: https://www.learnjsonschema.com/2020-12/ ![alt text](image.png) is helpful